packages/@react-spectrum/combobox/test/ComboBox.test.js

```diff
@@ -3602,26 +3668,6 @@ describe('ComboBox', function () {
         items = within(listbox).getAllByRole('option');
         expect(items).toHaveLength(1);
         expect(items[0]).toHaveTextContent('Aardvark');
-
-        act(() => {
-          triggerPress(items[0]);
-          jest.runAllTimers();
-        });
-
-        progressSpinner = getByRole('progressbar');
-        expect(progressSpinner).toBeTruthy();
-        expect(progressSpinner).toHaveAttribute('aria-label', 'Loading...');
-
-        await waitFor(() => expect(load).toHaveBeenCalledTimes(3));
-        expect(load).toHaveBeenLastCalledWith(
-          expect.objectContaining({
-            'filterText': 'Aardvark'
-          })
-        );
-
-        expect(() => getByRole('progressbar')).toThrow();
-        expect(() => getByTestId('tray')).toThrow();
-        expect(button).toHaveAttribute('aria-labelledby', `${getByText('Combobox').id} ${getByText('Aardvark').id}`);
```

Comment:
removed for now cuz I couldn't get the triggerPress/userEvent.click/etc to work without it complaining about not wrapping it in an act (even though it is...)

Reply:
ah, probably because of the promise if i had to guess, those are tricky, i don't recall what you need to do off the top of my head :(